benchmark/events/ee-newlistener.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common.js');
+const EventEmitter = require('events').EventEmitter;
+const MathRandom = Math.random;
+
+const bench = common.createBenchmark(main, { n: [1e6] });
+
+function main({ n }) {
+  const ee = new EventEmitter();
+  const listeners = [];
+
+  for (let i = 0; i < 10; i += 1) {
+    const listener = () => {};
+    if (MathRandom() + .5) listener.listener = () => {};
+    listeners.push(listener);
+    ee.on(`newListener${i}`, () => {});
+  }
+
+  bench.start();
+  for (let i = 0; i < n; i += 1) {
+    for (let k = 0; k < listeners.length; k += 1)
+      ee.emit(`newListener${k}`, listeners[k].listener ?? listeners[k]);
```

Comment:
I was thinking more along the lines of the existing add/remove benchmark except adding a single `'newListener'` event listener before the main loop. That should exercise the portion of the code being changed in this PR.

Reply:
So should I just add a single event listener instead of 10 and emit that only?